docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -281,63 +281,39 @@ role ``meta/argument_specs.yml`` file. All fields are lower-case.
     * This should be ``main`` in the case of an unspecified entry point.
     * This will be the base name of the tasks file to execute, with no ``.yml`` or ``.yaml`` file extension.
 
-    :short_description:
+.. csv-table:: Role Argument Specification Fields :rst:dir:`csv-table`
```

Comment:
Looks like we mostly use rst tables
```
.. table::
   :class: documentation-table
```

However I kind of like the current documentation format. I'd be in favor of just adding the notes portion.

Reply:
i dont see a good way to add the diff columns (type/required/default) to the current format